docs/en/operations/settings/settings.md

```diff
@@ -3065,4 +3065,48 @@ SELECT
 FROM fuse_tbl
 ```
 
+## allow_experimental_database_replicated {#allow_experimental_database_replicated}
+
+Enables to create databases with [Replicated](../../engines/database-engines/replicated.md) engine.
+
+Possible values:
+
+-   0 — Disabled.
+-   1 — Enabled.
+
+Default value: `0`.
+
+## database_replicated_initial_query_timeout_sec {#database_replicated_initial_query_timeout_sec}
+
+Sets how long initial DDL query should wait for Replicated database to precess previous DDL queue entries in seconds.
+
+Possible values:
+
+-   Positive integer.
+-   0 — Unlimited.
+
+Default value: `300`.
+
+## database_replicated_ddl_output {#database_replicated_ddl_output}
```

Comment:
The setting is obsolete, it has been replaced with `distributed_ddl_output_mode` (#21535)